ale_linters/luau/selene.vim

```diff
@@ -0,0 +1,48 @@
+" Copied from ..\lua\selene.vim
+
+call ale#Set('luau_selene_executable', 'selene')
+call ale#Set('luau_selene_options', '')
+
+function! ale_linters#luau#selene#GetCommand(buffer) abort
+    return '%e' . ale#Pad(ale#Var(a:buffer, 'luau_selene_options'))
+    \   . ' --display-style=json -'
+endfunction
+
+function! ale_linters#luau#selene#Handle(buffer, lines) abort
+    let l:output = []
+
+    for l:line in a:lines
+        " as of version 0.17.0, selene has no way to suppress summary
+        " information when outputting json, so stop processing when we hit it
+        " (PR for this here: https://github.com/Kampfkarren/selene/pull/356)
```

Comment:
This was merged a while ago

Reply:
Not sure about that one, literally just copied over the lua linter and changed all lua references to luau since selene works for both.